docs/source/data/file-uploads.md

```diff
@@ -3,11 +3,11 @@ title: File uploads
 description: Enabling file uploads in Apollo Server
 ---
 
-> Note: As of Apollo Server 3.0, in order to allow the `graphql-upload` package to evolve and live on its own, uploads are no longer enabled by default and are not an integrated part of Apollo Server.
+You can add file upload support to Apollo Server via the third-party [`graphql-upload`](https://npm.im/graphql-upload) library. This package provides support for the `multipart/form-data` content-type.
 
-File uploads can be enabled by using the third-party [`graphql-upload`](https://npm.im/graphql-upload) package.  Using this package will introduce support for the `multipart/form-data` content-type.
+> **New in Apollo Server 3:** File uploads are no longer supported by the default `apollo-server` library. To enable file uploads with the `graphql-upload` package, instead use the `apollo-server-express` library (which supports greater customization) and follow the steps in this article.
```

Comment:
I mean it's new in Apollo Server 3 that there's no built in easy integration at all with any version... ie, yes, part of what's new is that you can't do it `apollo-server` at all, but it's also a bunch more steps to do it with `apollo-server-express`.